tests/mypy/configs/mypy-default.ini

```diff
@@ -7,8 +7,15 @@ disallow_any_generics = True
 check_untyped_defs = True
 no_implicit_reexport = True
 
+python_version = 3.10
```

Comment:
Adding the version was necessary because the output differs between Python versions:

- `List/Dict` vs `list/dict`/`Union` vs `|`: covered by the `force_*` settings below.
- `ellipsis` vs `EllipsisType`: covered by this setting